.github/workflows/pythonapp.yml

```diff
@@ -4,7 +4,6 @@ on:
   # quick tests for pull requests and the releasing branches
   push:
     branches:
-      - dev
```

Comment:
Hi @Nic-Ma, I also remove the quick tests from `dev` because they are very similar to the pre-merge tests, mostly duplicated except for the versioning and branch tags. removing them will make the entire ci/cd faster...

Reply:
Sure, sounds good to me.
Thanks.